pkg/apis/certmanager/v1/types_certificaterequest.go

```diff
@@ -33,6 +33,14 @@ const (
 	// Issued indicates that a CertificateRequest has been completed, and that
 	// the `status.certificate` field is set.
 	CertificateRequestReasonIssued = "Issued"
+
+	// Approved indicates that a CertificateRequest has been approved by the
+	// approver, and the CertificateRequest is ready for signing.
```

Comment:
Suggestion: `.. the CertificateRequest is ready for signing` - could we perhaps word this differently? I understand that in this case it will be the associated X.509 certificate that can now be signed, so maybe `the certificate is ready for signing` ?
(Same with `CertificateRequestReasonDenied`).